package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "gatsby-starter-default",
+  "description": "Gatsby default starter",
+  "version": "1.0.0",
+  "author": "Kyle Mathews <mathews.kyle@gmail.com>",
```

Comment:
Just a reminder, could you please clean this a bit?

Reply:
Of course! I missed that